### DISCUSSION TITLE
What is the status of this project?

### DISCUSSION THREAD

**Discussion by listepo** [vitusortner/floor#797] [Q&A]
The project has not been updated for a long time, will it still be supported?

**Comment by stephanmantel** [comment]
Hi there,

Thank you for your interest in this project and your question. It is true that development for this project has declined over the last year, which I regret. We appreciate all input and collaboration. It is still our ambition to keep Floor relevant and we intend to keep supporting it.

Thanks again for your attention and your patience.
